### DISCUSSION TITLE
404 There isn't a GitHub Pages site here

### DISCUSSION THREAD

**Discussion by jeanrichet** [community/community#69123] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I created a repository to publish my website. I expect it to be published on https://jeanrichet.github.io/essailurcy
Unfortunately i get the answer "There isn't a GitHub Pages site here"
I am unable to find thr "visit the site" button, as it itis suggested to click on.
Can you help?
Thanks
![thereisntaGitHubPagessiteHere](https://github.com/community/community/assets/44022507/4a03155e-b5ae-4a35-85c1-deb6ef1ec43d)

jean.richet@orange.fr

**Comment by seyLu** [comment]
#### Hi @jeanrichet,

You forgot to configure github pages on that repo:
- repo settings > pages
- configure deploy source (in this case, deploy from branch)
- configure deploy branch

![image](https://github.com/community/community/assets/98249191/f91fd2d0-18fd-4f2e-8b89-1b3f3189a658)
